File: data/src/main/java/com/ykb/app/cryptotrader/data/repo/AuthorityRepo.java
package com.ykb.app.cryptotrader.data.repo;

import com.ykb.app.cryptotrader.data.model.Authority;
import com.ykb.app.cryptotrader.data.model.User;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AuthorityRepo extends JpaRepository<Authority, String> {
}

File: library/data/data-auth/src/main/kotlin/com/ykb/app/cryptotrader/data/auth/repo/AuthorityRepo.kt
package com.ykb.app.cryptotrader.data.auth.repo

import com.ykb.app.cryptotrader.data.auth.model.Authority
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface AuthorityRepo : JpaRepository<Authority, String>